public/views/index.html

```diff
@@ -14,6 +14,15 @@
 
     <base href="[[.AppSubUrl]]/" />
 
+    <script>
```

Comment:
Make sure to include the `<script nonce="[[.Nonce]]">` things like the other inline script tags. Needed for Strict-CSP configuration which don't allow inline scripts without a nonce.